code/modules/reagents/machinery/chem_dispenser.dm

```diff
@@ -353,11 +352,14 @@
 	circuit = /obj/item/weapon/circuitboard/chemical_dispenser/industrial
 	level0 = list(
 		"acetone","aluminum","ammonia",
-		"copper","ethanol","hydrazine",
-		"iron","radium","sacid",
-		"hclacid","silicon","tungsten"
+		"copper","ethanol","hclacid",
+		"hydrazine","iron","phosphorus",
+		"potassium", "radium","sacid",
+		"silicon","sulfur","tungsten"
 	)
 
 	level1 = list("oil", "cryptobiolin", "sterilizine")
+	level2 = list("foaming_agent", "plasticide", "instant_ice")
+	level3 = list("pacid") //Now it has chems for what it's supposed to do, and worth upgrading past Picos
```

Comment:
```suggestion
	level3 = list("pacid", "mutagen", "potassium_chloride", "cryptobiolin") //Now it has chems for what it's supposed to do, and worth upgrading past Picos
```
as above

Reply:
Again, it's an **industrial** themed chem dispenser, why would you need to make a lethal injection chemical and antibiotics with it...?